### PR TITLE
adding all files, including game classes, main, and the RSpec to test…

### PR DESCRIPTION
**Pull request by gonjavi**
… methods

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,103 @@
+require './lib/classes.rb'
+  	
+RSpec.describe Check_win do
+	describe "#check win" do
+		win= Check_win.new
```
Comment (bellom) [inline spec/game_spec.rb]: There should be space around operator; 
`win= Check_win.new` ------Bad
`win = Check_win.new` ------Good
One of the Microverse guideline state; students should make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide

_Kindly apply this to the rest the operator in your code._

File: spec/game_spec.rb [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		it "it returns true if there is a victory" do 
+			board=["-","-","-","o","o","o","-","-","-"]
+			expect(win.check_win(board)).to eql(true)
+		end
+		it "it returns true if there is a victory" do
```
Comment (bellom) [inline spec/game_spec.rb:12]: Leave one empty line around it/specify blocks. This helps to separate the expectations from their conditional logic (contexts for instance).
Checkout this link for more clarification. :point_right: https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples

kindly apply to the rest of the `it` block in your code.

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @gonjavi,
Great work using `Git Flow` and awesome job writing the test, specs are failing if i break some code in tested class.
Just two things to do; kindly attend to my comments.

_Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you_

File: spec/game_spec.rb (hunk 1 above)
Comment (gonjavi) [reply spec/game_spec.rb]: perfect

File: spec/game_spec.rb (hunk 2 above)
Comment (gonjavi) [reply spec/game_spec.rb:12]: perfect

File: spec/game_spec.rb (hunk 1 above)
Comment (gonjavi) [reply spec/game_spec.rb]: specs are failing if i break some code in tested class? this part I don't get it

**Comment by gonjavi** [comment]
I fixed the requested, please check if it is ok. thanks

File: spec/game_spec.rb (hunk 1 above)
Comment (bellom) [reply spec/game_spec.rb]: 😁, am simply saying your code test is working fine.

**Comment by bellom** [comment]
Can you please leave space around the rest of the operators you are having inside the code, you did for some :slightly_smiling_face:

Am talking about the operators inside `it` blocks.

**Comment by gonjavi** [comment]
perfect, I fixed them

**Comment by Oghenebrume50** [comment]
Reviewing........

**Review by Oghenebrume50 — APPROVED** [review]
Hi, @gonjavi  everything looks good now this is approved :+1:

**Pull request merged**